app/models/award.rb

```diff
@@ -21,4 +21,11 @@ def needs_role_input?
     award_type == "acting" 
   end
 
+  def pulldown_options
```

Comment:
Doesn't depend on instance state (maybe move it to another class?) (UtilityFunction)